src/algorithms/expected_consensus/_index.md

```diff
@@ -182,7 +142,7 @@ The probability that two Tipsets with different blocks would have all the same t
 
 {{<label finality>}}
 ## Finality in EC
-EC enforces a version of soft finality whereby all miners at round N will reject all blocks that fork off prior to round N-F. For illustrative purposes, we can take F to be 500. While strictly speaking EC is a probabilistically final protocol, choosing such an F simplifies miner implementations and enforces a macroeconomically-enforced finality at no cost to liveness in the chain.
+EC enforces a version of soft finality whereby all miners at round N will reject all blocks that fork off prior to round N-F. For illustrative purposes, we can take F to be 900. While strictly speaking EC is a probabilistically final protocol, choosing such an F simplifies miner implementations and enforces a macroeconomically-enforced finality at no cost to liveness in the chain.
```

Comment:
dont forget to update specs-actors for this too or just file an issue ❤️ 
https://github.com/filecoin-project/specs-actors/blob/eac38c1d153cb985fa147c97113b28718361c90b/actors/builtin/miner/policy.go#L50

Reply:
https://github.com/filecoin-project/specs-actors/pull/392 definitely would have forgotten. Thanks for the reminder, hawkeye